lms/static/scripts/frontend_apps/components/SubmitGradeForm.js

```diff
@@ -86,7 +91,8 @@ const useFetchGrade = student => {
  */
 export default function SubmitGradeForm({ student }) {
   // State for loading the grade
-  const { grade, gradeLoading } = useFetchGrade(student);
+  const [fetchGradeError, setFetchGradeError] = useState('');
+  const { grade, gradeLoading } = useFetchGrade(student, setFetchGradeError);
```

Comment:
The source of truth needs to be SubmitGradeForm, not useFetchGrade. So to make this work, the setFetchGradeError setter gets passed to `useFetchGrade`.